docs/development.md

```diff
@@ -92,84 +93,143 @@ $ boilerplate/_lib/container-make
 
 ## How to run
 
-Regardless of how you choose to run the operator, before doing so you will need to install the `UpgradeConfig` CRD on your cluster:
+Regardless of how you choose to run the operator, before doing so ensure the `UpgradeConfig` CRD is present on your cluster:
 
 ```shell
 $ oc create -f deploy/crds/upgrade.managed.openshift.io_upgradeconfigs_crd.yaml
 ```
 
 ### Locally
 
-- Make sure you have the [operator-sdk](https://github.com/operator-framework/operator-sdk/releases) binary in your `$PATH`.
+- Make sure you have the [operator-sdk](#operator-sdk) `$PATH`.
 
 - If you are not using an account that has `cluster-admin` privileges, you will need to [elevate permissions](https://github.com/openshift/ops-sop/blob/master/v4/knowledge_base/manage-privileges.md) to possess them.
 
-- Create a project for the operator to run inside of.
+- Create a project for the operator to run inside of:
 
 ```
-$ oc new-project managed-upgrade-operator
+$ oc new-project test-managed-upgrade-operator
 ```
 
 - Run the operator via the Operator SDK:
 
 ```
-$ OPERATOR_NAMESPACE=managed-upgrade-operator operator-sdk run --local --watch-namespace=""
+$ OPERATOR_NAMESPACE=test-managed-upgrade-operator operator-sdk run --local --watch-namespace=""
 ```
 
 (`make run` will also work here)
 
-- Trigger a reconcile loop by applying an `upgradeconfig` CR with your desired specs.
+- Trigger a reconcile loop by applying an `upgradeconfig` CR with your desired specs:
 
 ```shell
 $ oc apply -f test/deploy/upgrade.managed.openshift.io_v1alpha1_upgradeconfig_cr.yaml
 ```
 
 ### Remotely
 
-- Create a project for the operator to run inside of.
+- Build the image. In this example, we will use [Quay](http://quay.io/) as the container registry for our image:
 
 ```shell
-$ oc new-project managed-upgrade-operator
+$ operator-sdk build quay.io/<QUAY_USERNAME>/managed-upgrade-operator:latest
 ```
 
-- Build the image. In this example, we will use [Quay](http://quay.io/) as the container registry for our image.
+- Push the image to the registry:
+
+---
+
+**NOTE**
+
+Refer to [quay.md](./quay.md) for quay registry setup and podman login.
+
+---
 
 ```shell
-$ operator-sdk build quay.io/myuser/managed-upgrade-operator:latest
+podman push quay.io/<QUAY_USERNAME>/managed-upgrade-operator:latest
 ```
 
-- Push the image to the registry.
+- Login to `oc` [as admin](https://github.com/openshift/ops-sop/blob/master/v4/howto/break-glass-kubeadmin.md#for-clusters-with-public-api) 
+  
+- If MUO is already deployed on the cluster in `openshift-managed-upgrade-operator` namespace, scale the deployment down to 0:
 
 ```shell
-podman push quay.io/myuser/managed-upgrade-operator:latest
+oc scale deployment managed-upgrade-operator -n openshift-managed-upgrade-operator --replicas=0
```

Comment:
This presumes other MUO users might already be running MUO in the `openshift-managed-upgrade-operator` namespace, which may not be correct (they may be running on a plain OCP cluster). 

Perhaps just a "ensure no other instances of `managed-upgrade-operator` are actively running on your cluster, as they may conflict" will suffice.